docs/technical_reference/unit_models/zero_order_unit_models/constructed_wetlands_zo.rst

```diff
@@ -8,8 +8,6 @@ See documentation for :ref:`single-input, single-output Helper Methods<siso_meth
 
 Electricity Consumption
 -----------------------
-This unit does not include energy consumption.
-
```

Comment:
I think for now we can just keep the section mentioning exclusion of energy consumption. Otherwise, we'd want to change the automation script to not write this section at all if there is no energy consumption to consider. Perhaps that could go in the follow-up that you mentioned.

Reply:
I would also rather keep these lines, FWIW. We just shouldn't remove them *and* keep the header.

If need, I'm happy to look at the documentation generation code and offer suggestion if this isn't already automated.

Reply:
@adam-a-a I actually didn't change anything about constructed wetlands, so I think it just might not have been updated b4. But essentially this is happening because the entry for column G of the excel (energy) is just a "--", which is also what we have for the feed_zo. So I think we could easily remove these lines _and_ the header by adding " and not (zo_name_list[i] == "constructed_wetlands_zo")" to line 340 of the auotmation code. But like was said, we can just address that in follow-up PR. Everything should be up-to-date now!